pkgs/applications/editors/vim/plugins/avante-nvim/default.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  vimUtils,
+}:
+
+let
+  version = "0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "yetone";
+    repo = "avante.nvim";
+    rev = "refs/tags/v${version}";
```

Comment:
is it worth using the last commit like the other vim plugins do?

Reply:
Indeed, that will help since the latest release is broken.